src/Surfnet/ServiceProviderDashboard/Application/CommandHandler/Entity/EntityChangeRequestCommandHandler.php

```diff
@@ -95,7 +92,7 @@ public function handle(PublishProductionCommandInterface $command)
                     ),
                     [$response]
                 );
-                $this->flashBag->add('error', 'entity.edit.error.publish');
+                $this->requestStack->getSession()->getFlashBag()->add('error', 'entity.edit.error.publish');
```

Comment:
Maybe add a convenience `getFlashbag` method?

Reply:
I feel more comfortable with this